packages/ra-core/src/actions/accumulateActions.ts

```diff
@@ -2,7 +2,19 @@ import { crudGetMany, crudGetMatching } from './dataActions';
 
 export const CRUD_GET_MANY_ACCUMULATE = 'RA/CRUD_GET_MANY_ACCUMULATE';
 
-export const crudGetManyAccumulate = (resource, ids) => ({
+export const crudGetManyAccumulate = (
```

Comment:
You will need to export the return type for each action creator, because it will be used in the reducers.